org/retroActionItems.ts

```diff
@@ -0,0 +1,36 @@
+import { peril } from "danger"
+import { Issues } from "github-webhook-event-types"
+import { IncomingWebhookSendArguments, MessageAttachment } from "@slack/client"
+
+/**
+ * When an issue has been labelled RFC, then trigger the scheduler
+ * to send reminders about the issue into our slack.
+ */
```

Comment:
Comment needs updating, ideally with a link to your RFC?

Reply:
Good to go after ^

Reply:
🙌 merging on green then